### PR TITLE
[NCL-8053] Use OidcClient instead of handwritten keycloak client

### PR DESCRIPTION
**Pull request by thescouser89**

### CODE REVIEW

File: src/main/java/org/jboss/pnc/cleaner/orchApi/OrchClientProducer.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 package org.jboss.pnc.cleaner.orchApi;
 
 import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.RequestScoped;
```
Comment (vibe13) [inline src/main/java/org/jboss/pnc/cleaner/orchApi/OrchClientProducer.java:21]: Not used?

**Review by vibe13 — APPROVED** [review]
@thescouser89 Wow that is a nice cleanup indeed! Great work, thanks for catching this!

File: src/main/java/org/jboss/pnc/cleaner/orchApi/OrchClientProducer.java (hunk 1 above)
Comment (thescouser89) [reply src/main/java/org/jboss/pnc/cleaner/orchApi/OrchClientProducer.java]: Thanks! Good catch. Removed in the rebase.

**Comment by thescouser89** [comment]
Thank you @vibe13 ! <3

**Pull request merged**